manifest-build-tools/application/update_rackhd_version.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python
+
+"""
+usage:
+./on-tools/manifest-build-tools/HWIMO-BUILD on-tools/manifest-build-tools/application/update_rackhd_version.py \
+--manifest-repo-dir build-manifests \
+--builddir b \
+--git-credential https://github.com/PengTian0,GITHUB \
+                     
+The required parameters:
+manifest-repo-dir: The directory of manifest repository
+builddir: The destination for repositories stored.
+git-credential: url, credentials pair for the access to github repos
+
+The optional parameter:
+is-official-release: if true, this release is official, the default value is false
+
+"""
+import argparse
+import sys
+import os
+import shutil
+from RepositoryOperator import RepoOperator
+from reprove import ManifestActions
+from version_generator import VersionGenerator
+from common import *
+
+import deb822
+import subprocess
+
+class UpdateRackhdVersion(object):
```

Comment:
a class name is better to be a noun. and function name can be a verb.
how about RackhdVersionUpdater or RackhdDebianControlUpdater ?
and comment the summarized purpose of this class is appreciated.